src/app/technical-record/brakes/brakes-psv/brakes-psv.component.html

```diff
@@ -0,0 +1,101 @@
+<dl class="govuk-summary-list">
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Brake code</dt>
+    <dd id="psv-brakes-brakeCode" class="govuk-summary-list__value">
```

Comment:
needs to be prepended by test-* if this is required by automation testers, please check with Dragos